sdk/core/Azure.Core.TestFramework/src/ProxyTransport.cs

```diff
@@ -118,10 +118,11 @@ private void RedirectToTestProxy(HttpMessage message)
                     case EntryRecordModel.Record:
                         break;
                     case EntryRecordModel.RecordWithoutRequestBody:
-                        message.Request.Content = null;
+                        message.Request.Headers.Add("x-recording-skip", "request-body");
```

Comment:
Curious: have we used this approach with adding headers to modify the recordings elsewhere?
Nit: `x-ms-` instead of `x-`?
Nit: put id string for Azure SDK Test Framework somewhere in the header name as well so if this is accidentally logged, folks would have a better chance of figuring out where it came from?

Reply:
These are header keys defined by the Test Proxy. The convention is that that they start with "x-recording". Any header keys using this convention will be interpreted by the Test Proxy and then stripped out from the request so they wouldn't be sent to the downstream recipient (i.e the service).

Reply:
Sounds good!